asv_bench/benchmarks/indexing.py

```diff
@@ -290,12 +290,24 @@ def setup(self):
         self.dti = dti
         self.dti2 = dti2
 
+        index = np.random.choice(dti, 10000, replace=True)
```

Comment:
Could you make this a new `class` benchmark and make this deterministic (i.e. no random)